src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputLineLengthIgnoringImportStatements.java

```diff
@@ -0,0 +1,22 @@
+/*
+LineLength
+fileExtensions = (default)all files
+ignorePattern = (import|//).*
+max = 58
+
+
+*/
+
+// violation below 'longer than 58 characters (found 71)'
+       package com.puppycrawl.tools.checkstyle.checks.sizes.linelength;
```

Comment:
instead of allowing comments `ignorePattern = (import|//).*`
can you expand `package` with extra spaces 
` package com.puppycrawl.tools.checkstyle.checks.sizes.                                          linelength;`
and make limit of line  longer that any `// violation` comment.

Reply:
Done